resources/postgres-secret.yaml.example

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres
+type: Opaque
+stringData:
+  password: password
+  dsn: "postgresql://postgres:password@localhost/postgres"
```

Comment:
Perfect gitops world we'd have a separate repo for kube artifacts, and a Secret there or I guess via some vault solution. That might be possible if we move Sippy to the DPCR cluster. For now, what do we think about manually creating the Secret?

Reply:
Yea I was manually creating it on the cluster (with a different password), I just had this here as an example